recipes/cpycppyy/meta.yaml

```diff
@@ -29,7 +29,9 @@ requirements:
 
 test:
   commands:
-    - ls {{ SP_DIR }}/libcppyy.cpython*.so
+    # This does not work as SP_DIR is the directory at build time.
+    # - ls {{ SP_DIR }}/libcppyy.cpython*.so
```

Comment:
use `$SP_DIR` instead of the jinja variable

Reply:
I see. I had understood the documentation that these variables are only available at build time.